pkg/gardenadm/translator/static_pod.go

```diff
@@ -0,0 +1,140 @@
+// SPDX-FileCopyrightText: 2025 SAP SE or an SAP affiliate company and Gardener contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package translator
```

Comment:
The package name `translator` is not quite self-explanatory. What about `staticpod(s)`?